internal/pkg/manifest/workload.go

```diff
@@ -743,7 +756,7 @@ func RedirectPlatform(os, arch, wlType string) (platform string, err error) {
 	if wlType == RequestDrivenWebServiceType && os == OSWindows {
 		return "", errAppRunnerInvalidPlatformWindows
 	}
-	// All architectures must be 'amd64' (the only one currently supported); leave OS as is.
+	// All architectures default to 'amd64' (though 'arm64' is now also supported); leave OS as is.
 	// If a string is returned, the platform is not the default platform but is supported (except for more obscure platforms).
 	return platformString(os, ArchAMD64), nil
```

Comment:
like chatted offline let's change this to be x86_64 instead of amd64 as default arch to be closer to our service values and educate users